packages/react-components/react-charts-preview/library/src/components/CommonComponents/ResponsiveContainer/useResponsiveContainerStyles.styles.ts

```diff
@@ -0,0 +1,33 @@
+import { makeStyles, mergeClasses } from '@griffel/react';
```

Comment:
Import these from @fluentui/react-components rather than from griffel directly.

Reply:
I see all react components are importing these from griffel directly.

Reply:
I think importing from `@griffel` is the correct behavior for components. This avoids circular deps as `@fluentui/react-components` will import this package.